scripts/actions/add-files-to-translation-queue.js

```diff
@@ -128,7 +133,17 @@ const main = async () => {
   const queue = await getTranslations({
     status: STATUS.PENDING,
   });
-  const fileDataToAddToQueue = translationDifference(queue, includedFiles);
+
+  const filterByLocaleOption = (file) => {
+    if (localeOption) {
+      return file.locale === localeOption;
+    } else return;
```

Comment:
in the case a user doesn't supply a locale, you want everything returned.

i think this need to be a truthy valued returned by default.